### PR TITLE
add dell inspiron 7460

### PR DESCRIPTION
**Pull request by glwbr**
- [x] Tested the changes in your own NixOS Configuration
- [x] Tested the changes end-to-end by using your fork of `nixos-hardware` and
      importing it via `<nixos-hardware>` or Flake input

### CODE REVIEW

File: dell/inspiron/7460/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  };
+
+  services = {
+    thermald = {
+      enable = lib.mkDefault true;
+    };
+
+    fwupd =	{
+			enable = lib.mkDefault true;
+		};
```
Comment (Mic92) [inline dell/inspiron/7460/default.nix:29]:
```suggestion
    fwupd =	{
	  enable = lib.mkDefault true;
    };
```

File: dell/inspiron/7460/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  imports = [
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+    ../../../common/gpu/intel/kaby-lake
+    ../../../common/gpu/nvidia/maxwell
+    ../../../common/gpu/nvidia/prime.nix
+  ];
+
+  # Essential Firmware
+  hardware.enableRedistributableFirmware = mkDefault true;
```
Comment (Mic92) [inline dell/inspiron/7460/default.nix:15]: This is also part of nixos-generate-config through `imports = [ (modulesPath + "/installer/scan/not-detected.nix") ];`

Comment (Mic92) [reply dell/inspiron/7460/default.nix:15]:
```suggestion
```

**Comment by glwbr** [comment]
I suspect there's an issue with the spacing, so I defined the service options inline.

**Pull request merged**